restli-tools/src/main/java/com/linkedin/restli/tools/snapshot/check/PegasusSchemaSnapshotCompatibilityChecker.java

```diff
@@ -345,4 +402,53 @@ private static void help()
     }
     return null;
   }
+
+  private static List<SchemaAnnotationHandler> getAnnotationHandlers(String handlerJarPaths, String classNames)
```

Comment:
These code is still a lot similar to `SchemaAnnotationValidatorCmdLineApp`, should DRY

Reply:
Yes, moved this one to a util class which can be used by both 'SchemaAnnotationValidatorCmdLineApp' and 'PegasusSchemaSnapshotCompatibilityChecker'.